-------------------------------------------------------------------------------
Test set: InvoiceServiceTest
-------------------------------------------------------------------------------
Tests run: 3, Failures: 0, Errors: 3, Skipped: 0, Time elapsed: 0.455 sec <<< FAILURE! - in InvoiceServiceTest
testA_createInvoice_Successfull(InvoiceServiceTest)  Time elapsed: 0.157 sec  <<< ERROR!
org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
	at InvoiceServiceTest.testInvoices(InvoiceServiceTest.java:63)
Caused by: org.hibernate.exception.GenericJDBCException: Error calling Driver#connect
	at InvoiceServiceTest.testInvoices(InvoiceServiceTest.java:63)
Caused by: java.sql.SQLException: Could not connect: Access denied for user 'root'@'localhost' (using password: YES)
	at InvoiceServiceTest.testInvoices(InvoiceServiceTest.java:63)
Caused by: org.mariadb.jdbc.internal.common.QueryException: Could not connect: Access denied for user 'root'@'localhost' (using password: YES)
	at InvoiceServiceTest.testInvoices(InvoiceServiceTest.java:63)

testB_readInvoices_Successfull(InvoiceServiceTest)  Time elapsed: 0.144 sec  <<< ERROR!
org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
	at InvoiceServiceTest.testInvoices(InvoiceServiceTest.java:63)
Caused by: org.hibernate.exception.GenericJDBCException: Error calling Driver#connect
	at InvoiceServiceTest.testInvoices(InvoiceServiceTest.java:63)
Caused by: java.sql.SQLException: Could not connect: Access denied for user 'root'@'localhost' (using password: YES)
	at InvoiceServiceTest.testInvoices(InvoiceServiceTest.java:63)
Caused by: org.mariadb.jdbc.internal.common.QueryException: Could not connect: Access denied for user 'root'@'localhost' (using password: YES)
	at InvoiceServiceTest.testInvoices(InvoiceServiceTest.java:63)

testC_readInvoiceById_Successfull(InvoiceServiceTest)  Time elapsed: 0.152 sec  <<< ERROR!
org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
	at InvoiceServiceTest.testInvoices(InvoiceServiceTest.java:63)
Caused by: org.hibernate.exception.GenericJDBCException: Error calling Driver#connect
	at InvoiceServiceTest.testInvoices(InvoiceServiceTest.java:63)
Caused by: java.sql.SQLException: Could not connect: Access denied for user 'root'@'localhost' (using password: YES)
	at InvoiceServiceTest.testInvoices(InvoiceServiceTest.java:63)
Caused by: org.mariadb.jdbc.internal.common.QueryException: Could not connect: Access denied for user 'root'@'localhost' (using password: YES)
	at InvoiceServiceTest.testInvoices(InvoiceServiceTest.java:63)

